www/js/metrics.js

```diff
@@ -1026,23 +1003,13 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
   $scope.checkCalorieCardExpanded = function() {
         return ($scope.expandedc)? "icon ion-chevron-up" : "icon ion-chevron-down";
   }
-  var expandFootprintDOM = function() {
-    var div = document.getElementById('dashboard-footprint');
-    if (div.style.height == '470px'){ 
-      div.style.height = '140px'
-      $ionicScrollDelegate.scrollTop()
-    } else { 
-      div.style.height = '470px'
-    }
+
+  $scope.changeFootprintCardHeight = function() {
+        return ($scope.expandedf)? "expanded-footprint-card" : "small-footprint-card";
   }
-  var expandCalorieDOM = function() {
```

Comment:
is it because it is using the dom directly?
I think that was one of my comments - so if that is why and you are using CSS instead, that's great!